src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1066,8 +1066,6 @@ void setOptimalIndex(int newOptimalIdx)
          */
         void rtcpTransform(ByteArrayBuffer baf)
         {
-            SeenFrame maxSentFrame = this.maxSentFrame;
-
```

Comment:
tsTranslation is conceptually final, so it can't be changed to null (it's not marked as such because we're reusing/resetting the SeenFrames). However, if we don't store the frame in a local variable, next time we check the frame might have changed to something with tsTranslation == null.